leases/utils.py

```diff
@@ -43,6 +45,52 @@ def calculate_season_end_date(lease_end: date = None) -> date:
     return default
 
 
+def calculate_winter_season_start_date(lease_start: date = None) -> date:
+    """Return the date when the winter season starts
+
+    If the current date is after the season end date,
+    it returns next year's default start date.
+
+    Winter Storage Leases always start on 15.9 the earliest
+    """
+    today = date.today()
+    default = date(day=15, month=9, year=today.year)
+
+    if lease_start:
+        return default.replace(year=lease_start.year)
+
+    # If the current day is before the winter season ends,
+    # the start day will be on the previous year
+    if today < date(day=10, month=6, year=today.year):
```

Comment:
I think this should be `<=` 🤔

Reply:
So on 10.6.2020 the start date will be 10.6.2021?

Reply:
on 10.6.2020 the start will be 15.9.2019

Reply:
Ohhh yeah, yeah, okay.